doc/governance/accept_recovery.rst

```diff
@@ -27,31 +27,57 @@ A member proposes to recover the network and other members can vote on the propo
 
 .. code-block:: bash
 
-    $ ccf_cose_sign1 --ccf-gov-msg-type proposal --ccf-gov-msg-created_at `date -uIs` --signing-key member1_privk.pem --signing-cert member1_cert.pem --content transition_service_to_open.json | \
-      curl https://<ccf-node-address>/gov/proposals --cacert service_cert.pem --data-binary @- -H "content-type: application/cose"
+    $ ccf_cose_sign1 \
+      --ccf-gov-msg-type proposal \
+      --ccf-gov-msg-created_at `date -uIs` \
+      --signing-key member1_privk.pem \
+      --signing-cert member1_cert.pem \
+      --content transition_service_to_open.json \
+    | curl https://<ccf-node-address>/gov/members/proposals:create?api-version=2023-06-01-preview \
+      --cacert service_cert.pem \
+      --data-binary @- \
+      -H "content-type: application/cose"
     {
-        "ballot_count": 0,
-        "proposal_id": "1b7cae1585077104e99e1860ad740efe28ebd498dbf9988e0e7b299e720c5377",
-        "proposer_id": "d5d7d5fed6f839028456641ad5c3df18ce963bd329bd8a21df16ccdbdbba1eb1",
-        "state": "Open"
+        "ballotCount": 0,
+        "proposalId": "1b7cae1585077104e99e1860ad740efe28ebd498dbf9988e0e7b299e720c5377",
+        "proposerId": "d5d7d5fed6f839028456641ad5c3df18ce963bd329bd8a21df16ccdbdbba1eb1",
+        "proposalState": "Open"
     }
```

Comment:
While here I decided these example lines were getting very hard to read, so reformatted each argument onto its own line. This should make _future_ diffs more readable, though makes this diff even messier.

The real content change in this (and all of the similar instances) is replacing something like `/gov/proposals` with `/gov/members/proposals:create?api-version=2023-06-01-preview`, and updating the example response to show the new schema.